### PR TITLE
release notes for Jenkins 3.5.4-2.150.1

### PR DESCRIPTION
**Pull request by kvish**
## Description
DCOS-49681

## Urgency
- [ ] Blocker 
- [ ] High
- [ X] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by kvish** [comment]
copied all files over from previous jenkins release, only difference is the release notes file for this version

File: pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+excerpt: Discover the new features, updates, and known limitations in this release of the Jenkins Service
+enterprise: false
+--- 
+
+Jenkins 3.5.4-2.150.1 was released on March 11, 2019.
+
+# Release Notes for Jenkins Service version 3.5.3-2.150.1
+
+## Improvements
+- Update to version 2.150.1 of Jenkins.
```
Comment (anithamuthu79) [inline pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md]: Rephrase line #15:
- Update to version 2.150.1.

File: pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Add strict mode support for Jenkins 2.150.1
+
+## Known Limitations
+- Docker Build and Publish plugins will need to be manually updated to restore functionality here
```
Comment (anithamuthu79) [inline pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md]: Rewrite line #19:
Docker build and publish plugins will be manually updated to restore the Jenkins functionality

@kvish Please check if the above rewrite matches to the context of replacing `here` with `Jenkins`.

File: pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Jenkins will store user records in a slightly different format which prevents downgrades: See[here](https://jenkins.io/security/advisory/2018-12-05/#SECURITY-1072)
```
Comment (anithamuthu79) [inline pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md]: Rewrite `See here` with the following:

Read more information about [forced migration of user records](https://jenkins.io/security/advisory/2018-12-05/#SECURITY-1072) documentation.

File: pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md (hunk 2 above)
Comment (kvish) [reply pages/services/jenkins/3.5.4-2.150.1/release-notes/index.md]: changed to plugin functionality

**Pull request merged**